curriculum/challenges/english/07-scientific-computing-with-python/learn-the-bisection-method-by-finding-the-square-root-of-a-number/65ef1be2b958cabf712296cd.md

```diff
@@ -9,7 +9,9 @@ dashedName: step-11
 
 Now you'll repeatedly narrow down the interval by finding the midpoint of the current interval and comparing the square of the midpoint with the target value.
 
-For that, inside the `else` block, create a `for` loop that runs up to `max_iterations` times.
+For that, inside the `else` block, create a `for` loop that runs up to `max_iterations` times. To iterate you can use range function, it returns a sequence of numbers starting from 0 and stops before a specified number.
+
+You should use `_` as a loop variable. The `_` acts as a placeholder and is useful when you need to use a variable but don't actually need its value.
```

Comment:
```suggestion
For that, inside the `else` block, create a `for` loop that runs up to `max_iterations` times.

For your loop, use `range` function, which generates a sequence of numbers you can iterate over. The syntax is `range(start, stop, step)`, where `start` is the starting integer (inclusive), `stop` is the last integer (not inclusive), and `step` is the difference between a number and the previous one in the sequence.

Also, use `_` as a loop variable. The `_` acts as a placeholder and is useful when you need to use a variable but don't actually need its value.
```

Reply:
@Dario-DC thanks for suggestions. Updated PR.